LEAF_Request_Portal/sources/Group.php

```diff
@@ -269,54 +269,82 @@ public function getMembers($groupID, bool $searchDeleted = false): array|string
      *
      * @return string
      */
-    public function addMember($member, $groupID): string
+    public function addMember($member, $groupID): array
     {
         $oc_db = new \Leaf\Db(\DIRECTORY_HOST, \DIRECTORY_USER, \DIRECTORY_PASS, \ORGCHART_DB);
         $employee = new \Orgchart\Employee($oc_db, $this->login);
 
         if (is_numeric($groupID)) {
-            $sql_vars = array(':userID' => $member,
-                ':groupID' => $groupID,);
+            $vars = array(':userID' => $member,
```

Comment:
Just a quick chuckle here.  Every time I see this in everyone's code, my knee-jerk reaction is "shouldn't this be a 'let'?"  Takes me a second to switch from js to php.  And then I had the thought, it would be kinda funny to mess with future developers if we started using the variable name "$lets" for things like this.  

To be clear, no change needed.  Just thought I would share where my brain was going.  ;-)